### PR TITLE
Implement exception handling

### PR DESCRIPTION
**Pull request by Konti08**

### CODE REVIEW

File: utils/fileSystem.py [hunk 1]
```diff
@@ -194,5 +196,7 @@ def get_from_image_root(session_id):
                 images.append(os.path.join(image_path, f))
         else:
             pass
+    else:
+        raise Exception("Image directory does not exist")
```
Comment (Empiire00) [inline utils/fileSystem.py:200]: this function may throw an exception, which is not catched in the following files:
- action_scripts/changeFormat.py (line 14)
- action_scripts/convertToLowPoly.py (line 17)
- action_scripts/convertToPixelArt.py (line 16)
- action_scripts/igPanoSplit.py (line 14)
- action_scripts/resizeImage.py (line 14)
- dynamic_functions/dimensionFunctions.py (lines 9, 22, 35, 48)

**Review by Empiire00 — CHANGES_REQUESTED** [review]
Missing error handling in several files

**Comment by Konti08** [comment]
> Missing error handling in several files

Exceptions are handled only in the root methods as for the frontend it is not relevant which error occurred

File: utils/fileSystem.py (hunk 1 above)
Comment (Konti08) [reply utils/fileSystem.py:200]: I will add the error handling for the dimension functions

**Review by Empiire00 — APPROVED** [review]
LGTM

**Pull request merged**